docs/ERC721.md

```diff
@@ -236,63 +241,63 @@ end
 
 The following contract presets are ready to deploy and can be used as-is for quick prototyping and testing. Each preset includes a contract owner, which is set in the `constructor`, to offer simple access control on sensitive methods such as `mint` and `burn`.
 
-### ERC721_Mintable
+### ERC721_Mintable_Burnable
 
-The ERC721_Mintable preset offers a quick and easy setup for creating NFTs. The contract owner can create and destroy tokens with the `mint` and `burn` methods. 
+The `ERC721_Mintable_Burnable` preset offers a quick and easy setup for creating NFTs. The contract owner can create tokens with `mint`, whereas token owners can destroy their tokens with `burn`. 
 
-### ERC721_Pausable
+### ERC721_Mintable_Pausable
 
-The ERC721_Pausable preset creates a contract with pausable token transfers, minting, and burning. This preset proves useful for scenarios such as preventing trades until the end of an evaluation period and having an emergency switch for freezing all token transfers in the event of a large bug. In this preset, only the owner can pause/unpause the contract.
+The `ERC721_Mintable_Pausable` preset creates a contract that enables the contract owner to `mint` new tokens as well as `pause` and `unpause` token transfers. This preset proves useful for scenarios such as preventing trades until the end of an evaluation period and having an emergency switch for freezing all token transfers in the event of a large bug.
```

Comment:
I think this could be read as a bug in the preset

Reply:
Ahh I see what you mean. Good call